src/devices/mercator.ts

```diff
@@ -213,6 +214,43 @@ const definitions: Definition[] = [
             device.save();
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_agumlajc'}],
+        model: 'SSWMPIR-ZB',
+        vendor: 'Mercator Ikuü',
+        description: 'Combination Sensor',
+        fromZigbee: [fz.occupancy, fz.temperature, fz.humidity, fz.illuminance, fz.on_off, tuya.fz.datapoints],
```

Comment:
```suggestion
        fromZigbee: [tuya.fz.datapoints],
```

Please test if it also works with just this